tests/RuleSet/RuleSetsTest.php

```diff
@@ -65,6 +65,8 @@ public function testGetUnknownSetDefinition(): void
     public function testHasIntegrationTest(string $setDefinitionName): void
     {
         $setsWithoutTests = [
+            '@PER',
+            '@PER:risky',
```

Comment:
PER v1.0 is identical to PSR-12. Would it be OK to create integration tests after v1.1 is tagged, so that there are new rules, or should I copy tests for PSR-12?

Reply:
I'm OK with adding tests when updating PER later.

Reply:
Added some in #6547 (still draft, as v1.1 hasn't been released yet).